types/exchange/index.ts

```diff
@@ -0,0 +1,193 @@
+import { ApiRx } from "@cennznet/api";
```

Comment:
haven't we moved this file to `types.d.ts` already?

Reply:
ah yup, i will remove this file